ecf/scripts/gdas/wave/init/jgdas_wave_init.ecf

```diff
@@ -4,8 +4,7 @@
 #PBS -q %QUEUE%
 #PBS -A %PROJ%-%PROJENVIR%
 #PBS -l walltime=00:10:00
-#PBS -l place=vscatter,select=1:mpiprocs=128:ompthreads=1:ncpus=128
-#PBS -l place=excl
+#PBS -l place=vscatter,select=1:mpiprocs=11:ompthreads=1:ncpus=11
```

Comment:
excl or memory? Wei requested every job have one of either set. The waveinit job barely needs resources so maybe `mem=1GB`?

Reply:
700MB on wcoss1. I would suggest to start with 2GB

Reply:
Resolved in a1ca249a